users/templates/users_detail.html

```diff
@@ -61,6 +63,11 @@ <h1>
     {% trans 'go to the Django Admin site' %}.
   </div>
 
+  <div id="dialog_user_account_update" style="display: none" title="{% trans 'Are you sure' %}?" align="center">
+    {% trans 'Do you want to update' %} <br/> <b>{{ user_detail.email }} </b>
+    {% trans ' account' %}?
+  </div>
+
```

Comment:
Was there an issue for this popup? If so, it's not included in PR description.
It would be awesome, if changing user type from manager or admin to employee would prompt a special warning about projects though. Please consider it, if it's possible without some sophisticated mechanisms.

Reply:
But there's another issue: should it be possible to manually change status from employee to manager? Because then you have a manager that is not managing anything. Also, he will likely be changed back to an employee as soon as `remove_manager_from_project_action` is called.

Reply:
Ok, he doesn't change back apparently.